src/main/java/com/rcv/Tabulator.java

```diff
@@ -871,10 +849,7 @@ void doAudit(List<CastVoteRecord> castVoteRecords) {
   enum OvervoteRule {
     EXHAUST_IMMEDIATELY("exhaustImmediately"),
     ALWAYS_SKIP_TO_NEXT_RANK("alwaysSkipToNextRank"),
-    EXHAUST_IF_ANY_CONTINUING("exhaustIfAnyContinuing"),
```

Comment:
Do a project-wide search for all 3 of these modes and make sure you don't miss any. I see some config files that still use them, so we'll want to replace those.

Reply:
It was just the config that tests validation of invalid params. But I've updated it anyway for clarity.